docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,736 @@
+---
+title: "Build a Flutter App that Integrates with Viam"
+linkTitle: "Build a Flutter App"
+type: "docs"
+description: "Use Viam's Flutter SDK to build a custom mobile app to show your machines and their components."
+webmSrc: "/tutorials/flutter-app/demo.webm"
+mp4Src: "/tutorials/flutter-app/demo.mp4"
+videoAlt: "An example Viam-integrated Flutter app."
+tags: ["sdk", "flutter"]
+authors: ["Clint Purser"]
+languages: ["flutter"]
+viamresources: ["base"]
+level: "Intermediate"
+date: "2024-01-17"
+cost: "0"
+---
+
+<div class="td-max-width-on-larger-screens">
+ <div class="alignleft" style="max-width:250px; margin-right:40px">
```

Comment:
Looking at other tutorials, it doesn't seen necessary on them. The reason I included it here is because the bullet points don't align correctly--see also tables with unordered lists in them, where the text aligns instead of indenting to make room for the bullets. I did this as an interim good-enough instead of digging into the bullet bug. I'll make a ticket.